.circleci/config.yml

```diff
@@ -422,3 +433,14 @@ workflows:
             - "firefox stable group"
             - "chrome stable p2p"
             - "chrome stable group"
+  Release-candidate:
+    jobs:
+      - trigger-qe-tests:
+          filters:
+            tags:
+              only:
+                - /^\d+\.\d+\.\d+-rc\d+$/
```

Comment:
please include `/^\d+\.\d+\.\d+-rc\.\d+$/` in the matching tags as well.
So far we have been using `1.20.1-rc1` format which is not semvar compliant. We are going to move to using `1.20.1-rc.1` which is correct usage.

Reply:
same goes for beta and preview tags as well.

Reply:
Is this change happening @makarandp0?